test/connectors/test-sql-connector.js

```diff
@@ -101,6 +180,49 @@ TestConnector.prototype.createTable = function(model, cb) {
   });
 };
 
+TestConnector.prototype.getInsertedId = function(model, info) {
+  return info;
+};
+
+TestConnector.prototype.fromColumnValue = function(propertyDef, value) {
+  return value;
+};
+
+TestConnector.prototype.beginTransaction = function(isolationLevel, cb) {
+  var name = 'tx_' + transactionId++;
+  cb(null, new MockTransaction(this, name));
+};
+
+TestConnector.prototype.commit = function(tx, cb) {
+  tx.commit(cb);
+};
+
+TestConnector.prototype.rollback = function(tx, cb) {
+  tx.rollback(cb);
+};
+
 TestConnector.prototype.executeSQL = function(sql, params, options, callback) {
-  callback(null, []);
+  var transaction = options.transaction;
+  if (transaction && transaction.connector === this && transaction.connection) {
+    if (sql.indexOf('INSERT') === 0) {
```

Comment:
I don't like this check :( ... dont we have a tokenized version of the sql we can look at?